src/MCPClient/lib/clientScripts/archivematicaSetTransferType.py

```diff
@@ -35,4 +35,9 @@
     transferUUID = sys.argv[1]
     transferType = sys.argv[2]
 
-    Transfer.objects.filter(uuid=transferUUID).update(type=transferType)
+    transfer = Transfer.objects.get(uuid=transferUUID)
+    if transfer.type is not None:
+        sys.exit(0)
+
+    transfer.type = transferType
+    transfer.save()
```

Comment:
Hmm I'm not sure, should we consider a scenario where `type` is an empty string?